src/main/resources/errors.yaml

```diff
@@ -284,7 +284,11 @@ request-errors:
       Example map type definition:
```

Comment:
Should this also mention optional "service" parameter now that example shows it?